2022-06-02 16:05:01 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_CEP CHARACTER VARYING(8)": "'55031-430' (9)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(1, 'José', 'jose123', '123456', 'jose@jose.com', '123', '1991-01-01', '(21)123451234', 'Rua Maria José de Souza, nº 102, Cidade Alta, Caruaru, Caruaru', '55031-430') [22001-212]
2022-06-02 16:05:01 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_CEP CHARACTER VARYING(8)": "'38700-098' (9)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(2, 'Renato', 'RenatinhodeTere', '172839', 'renato@tere.com.br', '456789', '1975-12-01', '(21) 999994444', 'Travessa Sumbem, nº 259, Centro, Patos de Minas, MG', '38700-098') [22001-212]
2022-06-02 16:05:01 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_CEP CHARACTER VARYING(8)": "'77022-004' (9)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(3, 'Joca', 'Joca123456', '7894561432', 'joca@joquito.com.br', '456', '1980-05-15', '(21)111112222', 'Quadra 602 Sul Avenida LO 13, nº 749, Plano Diretor Sul, Palmas, TO', '77022-004') [22001-212]
2022-06-02 16:05:01 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_CEP CHARACTER VARYING(8)": "'60410-316' (9)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(4, 'Aminadabe', 'amina', 'dabe', 'amina@da.be', '555', '1995-03-21', '(11)0000077777', 'Travessa São Mateus, nº 499, Parreão, Fortaleza, CE', '60410-316') [22001-212]
2022-06-02 16:05:01 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_CEP CHARACTER VARYING(8)": "'29145-006' (9)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(5, 'Jumara', 'Jumar4', '123456', 'ju@gmail.com', '123456', '2005-10-12', '(99)999999999', 'Rua Pernambuco, nº 864, Santa Bárbara, Cariacica, ES', '29145-006') [22001-212]
2022-06-02 16:05:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(21)123451234' (13)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(1, 'José', 'jose123', '123456', 'jose@jose.com', '123', '1991-01-01', '(21)123451234', 'Rua Maria José de Souza, nº 102, Cidade Alta, Caruaru, Caruaru', '55031430') [22001-212]
2022-06-02 16:05:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(21) 999994444' (14)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(2, 'Renato', 'RenatinhodeTere', '172839', 'renato@tere.com.br', '456789', '1975-12-01', '(21) 999994444', 'Travessa Sumbem, nº 259, Centro, Patos de Minas, MG', '38700098') [22001-212]
2022-06-02 16:05:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(21)111112222' (13)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(3, 'Joca', 'Joca123456', '7894561432', 'joca@joquito.com.br', '456', '1980-05-15', '(21)111112222', 'Quadra 602 Sul Avenida LO 13, nº 749, Plano Diretor Sul, Palmas, TO', '77022004') [22001-212]
2022-06-02 16:05:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(11)0000077777' (14)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(4, 'Aminadabe', 'amina', 'dabe', 'amina@da.be', '555', '1995-03-21', '(11)0000077777', 'Travessa São Mateus, nº 499, Parreão, Fortaleza, CE', '60410316') [22001-212]
2022-06-02 16:05:40 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(99)999999999' (13)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(5, 'Jumara', 'Jumar4', '123456', 'ju@gmail.com', '123456', '2005-10-12', '(99)999999999', 'Rua Pernambuco, nº 864, Santa Bárbara, Cariacica, ES', '29145006') [22001-212]
2022-06-02 16:06:35 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "CLIENTE_TELEFONE CHARACTER VARYING(11)": "'(10000077777' (12)"; SQL statement:
INSERT INTO public.cliente (cliente_id, cliente_nome, cliente_usuario, cliente_senha, cliente_email, cliente_cpf, cliente_dt_nasc, cliente_telefone, cliente_end_completo, cliente_cep) VALUES(4, 'Aminadabe', 'amina', 'dabe', 'amina@da.be', '555', '1995-03-21', '(10000077777', 'Travessa São Mateus, nº 499, Parreão, Fortaleza, CE', '60410316') [22001-212]
2022-06-03 18:19:25 jdbc[25]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebServlet.doGet(WebServlet.java:129)
	at org.h2.server.web.WebServlet.doPost(WebServlet.java:166)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.serratec.trabalho.api.esquadrao6.security.JwtRequestsFilter.doFilterInternal(JwtRequestsFilter.java:69)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:204)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:183)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:354)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:267)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:890)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 57 more
2022-06-03 18:29:18 jdbc[26]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at com.zaxxer.hikari.pool.HikariPool.access$100(HikariPool.java:71)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:726)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 17 more
